manifests/init.pp

```diff
@@ -348,5 +353,13 @@
       Class['jenkins::firewall'] ->
         Anchor['jenkins::end']
   }
+
+  if $config_template {
+    Anchor['jenkins::begin'] ->
```

Comment:
I see that some of the existing code is setting a bad example but this ordering isn't needed as there is an existing relationship.